### PR TITLE
[fix] fix readme

### PR DESCRIPTION
**Pull request by liuwenran**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

table of content not added  when eg3d readme merged

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ Table of contents:
 
 &#8195;      [2.2.9. Text-to-Image example](#229-text-to-image)
 
+&#8195;      [2.2.10. Text-to-Image example](#2210-3d-aware-generation-eg3d)
```
Comment (LeoXing1996) [inline demo/README.md:33]:
```suggestion
&#8195;      [2.2.10. 3D-aware Generation (EG3D)](#2210-3d-aware-generation-eg3d)
```

Comment (liuwenran) [reply demo/README.md:33]: sorry, my mistake :0

**Review by LeoXing1996 — APPROVED** [review]

**Comment by zengyh1900** [comment]
Hi @liuwenran ,

- [x] you should base on `dev-1.x` rather than `1.x`, I have changed it for you.
- [ ] you should fix lint. 

After that, I think this pull request can be merged.

**Comment by liuwenran** [comment]
this bug is fixed.

**Pull request closed without merging**